core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -632,4 +635,5 @@ trait WhiskActionsApi
 
 private case class BlockingInvokeTimeout(activationId: ActivationId) extends TimeoutException
 private case class TooManyActionsInSequence() extends RuntimeException
+private case class NoComponentInSequence() extends RuntimeException
```

Comment:
debatable. IllegalStateException makes me think that I made a request at a bad time (bad state on the server side). these exception look more similar to IllegalArgumentException - I'm performing a request with arguments that don't fit the bill/contract. 

I don't feel strongly about this, so I can change them. what's it gonna be? @rr @markusthoemmes